.github/workflows/self-new-model-pr-caller.yml

```diff
@@ -0,0 +1,124 @@
+name: New model PR CI
+
+on:
+  pull_request:
+    paths:
+      - "src/transformers/models/**"
+
+env:
+  HF_HOME: /mnt/cache
+  TRANSFORMERS_IS_CI: yes
+  OMP_NUM_THREADS: 8
+  MKL_NUM_THREADS: 8
+  RUN_SLOW: yes
+  # For gated repositories, we still need to agree to share information on the Hub repo. page in order to get access.
+  # This token is created under the bot `hf-transformers-bot`.
+  HF_HUB_READ_TOKEN: ${{ secrets.HF_HUB_READ_TOKEN }}
+  SIGOPT_API_TOKEN: ${{ secrets.SIGOPT_API_TOKEN }}
+  TF_FORCE_GPU_ALLOW_GROWTH: true
+  RUN_PT_TF_CROSS_TESTS: 1
```

Comment:
Let's only do torch for now no?

Reply:
I'd like to keep `RUN_PT_TF_CROSS_TESTS=1` - if there's any changes made to an important with a TF port which require slow tests to be run, we want to make sure the models are still equivalent